docs/_layouts/default.html

```diff
@@ -12,9 +12,9 @@
 
     {% include nav-home.html %}
 
-    <div id="content">
+    <main id="content" role="main">
```

Comment:
ah, in v3 we had `html5shiv` take care of that, forgot we don't have that in v4 anymore.

i think it'd make sense to add the default block styling to reboot. i'll take care of that before merging this, assuming @mdo is ok with it